lib/compat/wordpress-6.1/block-template-utils.php

```diff
@@ -248,6 +248,136 @@ function gutenberg_get_block_template( $id, $template_type = 'wp_template' ) {
 	return apply_filters( 'get_block_template', $block_template, $id, $template_type );
 }
 
+/**
+ * Builds the title and description of a post specific template based on the underlying referenced post.
+ * Mutates the underlying template object.
+ *
+ * @access private
+ * @internal
+ *
+ * @param string            $post_type Post type e.g.: page, post, product.
+ * @param string            $slug      Slug of the post e.g.: a-story-about-shoes.
+ * @param WP_Block_Template $template  Template to mutate adding the description and title computed.
+ *
+ * @return boolean Returns true if the referenced post was found and false otherwise.
+ */
+function _gutenberg_build_title_and_description_for_single_post_type_block_template( $post_type, $slug, WP_Block_Template $template ) {
+	$post_type_object = get_post_type_object( $post_type );
+
+	$posts = get_posts(
+		array(
+			'name'      => $slug,
+			'post_type' => $post_type,
+		)
+	);
+	if ( empty( $posts ) ) {
+		$template->title = sprintf(
+			// translators: Represents the title of a user's custom template in the Site Editor referencing a post that was not found, where %1$s is the singular name of a post type and %2$s is the slug of the deleted post, e.g. "Not found: Page(hello)".
+			__( 'Not found: %1$s(%2$s)', 'gutenberg' ),
```

Comment:
Remove the `gutenberg` domain, as this will fallback to Core domain.

Reply:
I think the domain has to be removed in the backport PR, not the Gutenberg PR. This function will disappear from Gutenberg code when the minimum WP version for Gutenberg increases to 6.2